index.html

```diff
@@ -237,6 +238,49 @@ <h2 class="head-1" id="contact-me">Contact Me</h2>
         </form>
       </div>
     </footer>
+    <!-- codes for modal pop-up -->
+    <div class="modal-ui md-hide" id="modal-ui-wrapper">
+      <div class="modal-card" id="modal-card">
+        <div class="mc-titlebar" id="mctitlebar">
+          <h2 class="mc-title" id="mc-title">Tonic</h2>
+          <button type="button" class="mc-btn" id="mc-close"> &times;</button>
+        </div>
+        <ul class="inline-li project-summary">
+          <li class="client" id="mc-client">CANOPY</li>
+          <li class="type" id="mc-type">Back End Dev</li>
+          <li class="year" id="mc-year">2015</li>
+        </ul>
+        <div class="mc-snap-place">
+          <img class="mc-snapshoot" src="images/SnapshootD1.jpg" alt="Snapshot of Project Tonic" id="mc-img">
+        </div>
+        <div class="mc-l-block">
+          <div class="detail mc-l-block-l" id='mc-details'>
+            <p class="project-descriptions" id="mc-desc">
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+              Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a galley of type and scrambled it to make a 
+              type specimen book. It has survived not only five centuries, but also the leap 
+              into electronic typesetting, remaining essent
+            </p>
+          </div>
+          <div class="mc-l-block-r">
+            <ul class="inline-li">
+              <li class="tags mc-tags">html</li>
+              <li class="tags mc-tags">css</li>
+              <li class="tags mc-tags">javascripts</li>
+              <li class="tags mc-tags">github</li>
+              <li class="tags mc-tags">ruby</li>
+            </ul>
+            <div class="modal-btn-area">
+              <button class="btn btn-modal" type="button" onclick="location.href='https://angomranjan.github.io/Profile/'">See Live <img src="icons/Icon.png" alt="" class="btn-icn"></button>
+              <button class="btn btn-modal" type="button" onclick="location.href='https://github.com/AngomRanjan/Profile'">See Source <img src="icons/Vector.png" alt="" class="btn-icn"></button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>    
```

Comment:
- [ ] Please make sure the model is created using javascript dynamically, Right now you have created the model in HTML and replaced its content using javascript but it should be created dynamically.